index.bs

```diff
@@ -2916,6 +2916,246 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method ### {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(optional PublicKeyCredentialSignalOptions options = {});
```

Comment:
Could each report type have a separate method? ex `signalUnknownCredential` and `signalCurrentCredentials.`

This would make the autocomplete easier and explain to the developer what each call does more directly.

I don't think these two types of reports will actually be often sent together. A `UnknownCredentialIdReport` seems to mostly get sent after a non-valid (to the RP) `.get()` while `CurrentCredentialsReport` gets sent after a valid (to the RP) `.get().`

The client could also coalesce reports sent within a small time period together if modifying credentials on an authenticator is a heavy action.

Reply:
I agree with your reasoning. I split the PR into three signal* methods, which should hopefully alleviate the confusion with the "signal" attribute and improve autocomplete. Even if a developer wanted to send multiple reports together, nothing is stopping them from calling multiple signal* methods!